src/frontend/views/accessibility.njk

```diff
@@ -0,0 +1,48 @@
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% extends "common/layout/base.njk" %}
+
+{% block beforeContent %}
+  <nav>
+    {{ govukBreadcrumbs(breadcrumbData) }}
+  </nav>
+{% endblock beforeContent %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">{{ organizationName }} Accessibility Statement</h1>
+
+      <div class="govuk-section" id="accessibilityStatement">
+        <h2 class="govuk-heading-l">Accessibility Statement</h2>
+        <div class="govuk-body">{{ organizationName }} is committed to making its {{ websiteOrApp }} accessible, in accordance with the Public Sector Bodies Websites (No. 2) Accessibility Regulations 2018.</div>
+        <div class="govuk-body">This accessibility statement applies to {{ statementScope }}.</div>
+      </div>
+
+      <div class="govuk-section" id="complianceStatus">
+        <h2 class="govuk-heading-l">Compliance Status</h2>
+        <div class="govuk-body">This website is fully compliant with the Web Content Accessibility Guidelines (WCAG) version {{ wcagVersion }} AA standard: <a href="https://www.w3.org/TR/WCAG21/" class="govuk-link">Web Content Accessibility Guidelines version 2.1</a></div>
+      </div>
+
+      <div class="govuk-section" id="preparation">
+        <h2 class="govuk-heading-l">Preparation of this accessibility statement</h2>
+        <div class="govuk-body">This statement was prepared on {{ statementDate }}.</div>
+        <div class="govuk-body">The statement was last reviewed on {{ statementLastReviewDate }}.</div>
+        <div class="govuk-body">The method used to prepare the statement: {{ statementPreparationMethod }}</div>
+      </div>
+
+      <div class="govuk-section" id="feedbackAndContact">
+        <h2 class="govuk-heading-l">Feedback and contact information</h2>
+        <div class="govuk-body">For any complaints please contact us <a href="https://signin.account.gov.uk/support" class="govuk-link">here </a> </div>
```

Comment:
This shouldn't go to the signin support.

"For any complaints please contact the Billing Transaction Monitoring Team via internal support".

We'll check if theres an appropriate email for our tech group...